documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/LoadBalancerPolicy.java

```diff
@@ -48,12 +48,19 @@ private LoadBalancerPolicy(Map<String, String> params) {
         }
 
         pattern = cluster + "/*/" + session;
-        loadBalancer = new LoadBalancer(cluster);
+        String type = params.get("type");
+        if (type == "adaptive") {
```

Comment:
Should use `equals()` instead of `==`.

Also consider flipping argument order to implicitly handle `type == null` in case no balancer type is present.

Reply:
Nevermind, I see this was fixed prior to me submitting this 🙂